{"name": "test_valid_sign_in[chrome]", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'test_rep'", "trace": "self = <Tests.UI_Test.test_login.TestLogin object at 0x000001CC02774A70>\nsetup_pages = (<selenium.webdriver.chrome.webdriver.WebDriver (session=\"02e0497cc9660f579d467876a3e26ad7\")>, <UI_Pages.loginPage.Log...ges.cartPage.CartPage object at 0x000001CC030358E0>, <UI_Pages.checkoutPage.CheckoutPage object at 0x000001CC03035370>)\nui_data = {'INVALID PASSWORD': 'Admin@123', 'PASSWORD': 'secret_sauce', 'Zipcode': '987867', 'firstName': 'Harry', ...}\nassertion_data = {'cart_heading': 'Shipping', 'expected_text': 'Epic sadface: Username and password do not match any user in this service', 'expected_url': 'https://www.saucedemo.com/inventory.html', 'product_name': 'Sauce Labs Backpack', ...}\nlogger_setup = None, request = <FixtureRequest for <Function test_valid_sign_in[chrome]>>\ninit_driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"02e0497cc9660f579d467876a3e26ad7\")>, browser = 'chrome'\n\n    @pytest.mark.run(order=2)\n    @pytest.mark.parametrize(\"browser\", [\"chrome\" ])\n    def test_valid_sign_in(self, setup_pages, ui_data, assertion_data, logger_setup,request, init_driver, browser):\n        logging.getLogger(\"root\").info(\"Starting test_valid_signin\")\n        allure.attach('Starting test_valid_signin', attachment_type=allure.attachment_type.TEXT)\n        driver, login_page, _, _ = setup_pages\n        logging.getLogger(\"root\").info(\"Logging with valid credentials\")\n        allure.attach('Logging with valid credentials', attachment_type=allure.attachment_type.TEXT)\n        self.login_and_assert(login_page, driver, ui_data, assertion_data)\n        allure.attach('Logging successfully', attachment_type=allure.attachment_type.TEXT)\n        logging.getLogger(\"root\").info(\"Ending test_valid_signin\")\n>       if request.node.test_rep[0].failed:\nE       AttributeError: 'Function' object has no attribute 'test_rep'\n\nTests\\UI_Test\\test_login.py:24: AttributeError"}, "attachments": [{"source": "24254d63-8697-40fe-b6f6-3a513cd985ba-attachment.txt", "type": "text/plain"}, {"source": "54211cd2-62af-44a1-83d5-fb801b9d030b-attachment.txt", "type": "text/plain"}, {"source": "4372274e-7b18-458e-9930-1588dbc82a9b-attachment.txt", "type": "text/plain"}, {"name": "log", "source": "fc43862b-ebe4-4bee-bf6c-801aecf86871-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "browser", "value": "'chrome'"}], "start": 1715251874817, "stop": 1715251875457, "uuid": "dcc32198-7148-4b7a-90d7-e89d63a18e61", "historyId": "f1aebfe96b81d111413432f15a02e381", "testCaseId": "c45d73e905ffe70f729e3c177209e9eb", "fullName": "Tests.UI_Test.test_login.TestLogin#test_valid_sign_in", "labels": [{"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver')"}, {"name": "parentSuite", "value": "Tests.UI_Test"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Feuji-006L-R"}, {"name": "thread", "value": "24248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.UI_Test.test_login"}]}